policyengine_us/variables/gov/states/id/tax/income/deductions/retirement_benefits/id_retirement_benefits_deduction.py

```diff
@@ -0,0 +1,36 @@
+from policyengine_us.model_api import *
+
+
+class id_retirement_benefits_deduction(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Idaho retirement benefits deduction"
+    unit = USD
+    documentation = "https://legislature.idaho.gov/statutesrules/idstat/title63/t63ch30/sect63-3022a/"
+    definition_period = YEAR
+    defined_for = StateCode.ID
+
+    def formula(tax_unit, period, parameters):
+        person = tax_unit.members
+        p = parameters(
+            period
+        ).gov.states.id.tax.income.deductions.retirement_benefits
+        filing_status = tax_unit("filing_status", period)
+
+        eligible_person = person(
+            "id_retirement_benefits_deduction_eligible_person", period
+        )
+        # Base retirement benefits deduction amount
+        base_amt = p.amount[filing_status]
+        # Social Security benefits received amount
+        ss_amt = person("social_security_retirement", period)
```

Comment:
```suggestion
        ss_amt = person("tax_unit_taxable_social_security", period)
```

Reply:
"social_security_retirement" is person level but "tax_unit_taxable_social_security" is tax unit level. why is it necessary to make this change?

Reply:
It is surrounding the "taxable" part, we can use `taxable_social_security` instead if we need the person level variable